browser-test/src/support/admin_programs.ts

```diff
@@ -597,34 +597,27 @@ export class AdminPrograms {
 
   /**
    * Edit basic block details and required and optional questions. Cannot handle more than one optional question.
-   * @deprecated prefer using {@link #editProgramBlockUsingSpec} instead. Be aware that the new method will place
-   * the optional question in the order as defined in the question array. The older method could only handle one
-   * optional question and forced it to be the first question on the list. Tests may need to be updated to handle
-   * a different question order.
+   * @deprecated prefer using {@link #editProgramBlockUsingSpec} instead. Be aware when migrating to
+   * editProgramBlockUsingSpec that this method always puts the optional question first in the block.
```

Comment:
suggestion (non-blocking):  When you say "this method" it isn't clear to me if that's `editProgramBlockUsingSpec` or `editProgramBlockWithOptional`.  How about "Be aware... that the following method always puts the optional question first in the block whereas `editProgramBlockUsingSpec` places it in the order of the question array."

Reply:
Thanks! Updated to just use both method names explicitly to avoid any confusion.